src/gpuarray/buffer_blas.h

```diff
@@ -115,15 +115,30 @@ GPUARRAY_PUBLIC int gpublas_hgemmBatch(
   float beta, gpudata **C, size_t *offC, size_t ldc,
   size_t batchCount, int flags);
 
-//TODO: float should be half
-GPUARRAY_PUBLIC int gpublas_hgemmStridedBatch(
+GPUARRAY_PUBLIC int gpublas_hgemm3D(
```

Comment:
I don't like much the new name. why that won't be a tensordot? 3d is the part I don't like too much.